website/citations/models.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+
+from modularodm import fields
+
+from framework.mongo import StoredObject
+
+
+class CitationStyle(StoredObject):
+
+    # Required Fields
+
+    # The name of the citation file, sans extension
+    _id = fields.StringField(primary=True)
+
+    # The full title of the style
+    title = fields.StringField()
+
+    # Datetime the file was last parsed
+    parsed = fields.DateTimeField(default=datetime.datetime.utcnow)
```

Comment:
For naming consistency with other models, could we change this to `date_parsed`?